src/frontend/packages/core/package.json

```diff
@@ -31,7 +31,7 @@
     "lint": "eslint 'src/**/*.{ts,tsx}'",
     "format:check": "prettier --check src",
     "format:write": "prettier --write src",
-    "build-theme": "cunningham -g css -o src"
+    "build-theme": "cunningham -g css,ts -o src"
   },
   "dependencies": {
     "@openfun/cunningham-react": "0.7.0",
```

Comment:
I think we should move this deps as `peerDependencies` then install it into `devDependencies` for the storybook.